guides/source/active_record_basics.md

```diff
@@ -71,105 +70,149 @@ Convention over Configuration in Active Record
 
 * `created_at` - Automatically gets set to the current date and time when the
   record is first created.
 * `updated_at` - Automatically gets set to the current date and time whenever
   the record is created or updated.
 * `lock_version` - Adds [optimistic
-  locking](https://api.rubyonrails.org/classes/ActiveRecord/Locking.html) to
-  a model.
+  locking](https://api.rubyonrails.org/classes/ActiveRecord/Locking.html) to a
+  model.
 * `type` - Specifies that the model uses [Single Table
   Inheritance](https://api.rubyonrails.org/classes/ActiveRecord/Base.html#class-ActiveRecord::Base-label-Single+table+inheritance).
-* `(association_name)_type` - Stores the type for
-  [polymorphic associations](association_basics.html#polymorphic-associations).
+* `(association_name)_type` - Stores the type for [polymorphic
+  associations](association_basics.html#polymorphic-associations).
 * `(table_name)_count` - Used to cache the number of belonging objects on
-  associations. For example, a `comments_count` column in an `Article` class that
-  has many instances of `Comment` will cache the number of existent comments
-  for each article.
+  associations. For example, a `comments_count` column in an `Article` class
+  that has many instances of `Comment` will cache the number of existing
+  comments for each article.
 
-NOTE: While these column names are optional, they are in fact reserved by Active Record. Steer clear of reserved keywords unless you want the extra functionality. For example, `type` is a reserved keyword used to designate a table using Single Table Inheritance (STI). If you are not using STI, try an analogous keyword like "context", that may still accurately describe the data you are modeling.
+NOTE: While these column names are optional, they are reserved by Active Record.
+Steer clear of reserved keywords when naming your table's columns. For example,
+`type` is a reserved keyword used to designate a table using Single Table
+Inheritance (STI). If you are not using STI, use a different word to accurately
+describe the data you are modeling.
 
 Creating Active Record Models
 -----------------------------
 
-When generating an application, an abstract `ApplicationRecord` class will be
-created in `app/models/application_record.rb`. This is the base class for all
-models in an app, and it's what turns a regular ruby class into an Active Record
-model.
+When generating a Rails application, an abstract `ApplicationRecord` class will
+be created in `app/models/application_record.rb`. The `ApplicationRecord` class
+inherits from [`ActiveRecord::Base`](https://api.rubyonrails.org/classes/ActiveRecord/Base.html) and it's what turns a regular Ruby class into
+an Active Record model.
 
-To create Active Record models, subclass the `ApplicationRecord` class and you're good to go:
+`ApplicationRecord` is the base class for all models in your app. To create
+Active Record models, subclass the `ApplicationRecord` class and you're good to
+go:
```

Comment:
Following that idea of trying to decouple a bit AR = model, maybe we can move `Active Record` to the first sentence:

```suggestion
`ApplicationRecord` is the base class for all Active Record models in your app.
To create a new model, subclass the `ApplicationRecord` class and you're good to
go:
```